src/proto/FunctionRpc.proto

```diff
@@ -648,3 +650,13 @@ message RpcHttp {
   map<string,NullableString> nullable_params = 21;
   map<string,NullableString> nullable_query = 22;
 }
+
+// Message representing Microsoft.Azure.WebJobs.ParameterBindingData
+// Used for hydrating SDK-type bindings in out-of-proc workers
+message BindingData
+{
+    string version = 1;
```

Comment:
We can add comments describing each field similar to other messages in this Proto file.